packages/core/src/pagination/pagination.element.ts

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2016-2021 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { baseStyles } from '@cds/core/internal';
+import { html, LitElement } from 'lit-element';
+import { styles } from './pagination.element.css.js';
+
+export const PAGINATION_LAYOUT = 'horizontal gap:md align:center';
+
+export class CdsPagination extends LitElement {
+  render() {
+    return this.hasAttribute('cds-layout')
+      ? html`<slot></slot>`
+      : html`<div cds-layout="${PAGINATION_LAYOUT}"><slot></slot></div>`;
```

Comment:
I get why we made the layout a constant but note that it also creates a reference in the template the Lit Element will try to keep track of. Since this is kind of a one-and-done application of the `PAGINATION_LAYOUT` const, I would consider wrapping it in a `cache()` directive. You will need to import it through lit-html, though, not lit-element.

https://lit-html.polymer-project.org/guide/writing-templates#caching-template-results:-the-cache-directive

Reply:
Any ideas on this comment?

Reply:
Is it possible to use that `cache` directive within `lit-element` templates? I tried it before. It seems to work only with `lit-html` templates.

Reply:
You should be able to...

https://lit-element.polymer-project.org/guide/templates#import-and-use-a-lit-html-directive